src/MigrationTools.ConsoleFull/Program.cs

```diff
@@ -55,7 +55,11 @@ public static async Task Main(string[] args)
 
                     // Core
                     services.AddSingleton<IMigrationClient, MigrationClient>();
-                   
+                    services.AddTransient<IMigrationClient, MigrationClient>();
```

Comment:
We adding MigrationClient as a Transient and Singleton?

Reply:
Fixing